src/components/side_panel/chart/radar_chart/radar_chart_design_panel.xml

```diff
@@ -0,0 +1,39 @@
+<templates>
+  <t t-name="o-spreadsheet-RadarChartDesignPanel">
+    <GeneralDesignEditor
+      figureId="props.figureId"
+      definition="props.definition"
+      updateChart="props.updateChart">
+      <t t-set-slot="general-extension">
+        <Section class="'pt-0'">
+          <t t-set-slot="title">Legend position</t>
+          <select
+            t-att-value="props.definition.legendPosition ?? 'top'"
+            class="o-input"
+            t-on-change="this.updateLegendPosition">
+            <option value="none">None</option>
+            <option value="top">Top</option>
+            <option value="bottom">Bottom</option>
+            <option value="left">Left</option>
+            <option value="right">Right</option>
+          </select>
```

Comment:
Could be worth to create a component, the same code is used 5 times. Can be done in another PR if you want.